tests/figures/chart/bar_chart_plugin.test.ts

```diff
@@ -59,7 +59,22 @@ describe("bar chart", () => {
 
   describe("Horizontal bar chart", () => {
     beforeEach(() => {
-      model = new Model();
+      model = new Model({
+        sheets: [
+          {
+            name: "Sheet1",
+            colNumber: 10,
+            rowNumber: 10,
+            rows: {},
+            cells: {
+              B1: { content: "first column dataset" },
+              B2: { content: "10" },
+              B3: { content: "11" },
+              B4: { content: "12" },
```

Comment:
avoid `beforeEach` with setup data please. Because it impacts all tests in the `describe` even if some/most of them do not need it. Tests setup should be minimal to test a given behavior.